docker_services_cli/cli.py

```diff
@@ -125,7 +131,7 @@ def __init__(self, filepath, verbose):
     "--filepath",
     "-f",
     required=False,
-    default=f"{get_python_lib()}/docker_services_cli/docker-services.yml",
+    default=f"{_get_module_path()}/docker-services.yml",
```

Comment:
@ppanero this here is related to your question regarding the path...
previously, the path to the `docker-services.yml` would be constructed via `distutils.sysconfig.get_python_lib()`, which will return the location where the Python source files are installed "normally":
```python
In [2]: get_python_lib()
Out[2]: '/home/mmoser/.local/share/virtualenvs/tu-data-3eTT44w9/lib/python3.8/site-packages'
```

However, if you install `docker-services-cli` in editable mode (`pip install -e /path/to/docker-services-cli`, then this would be obviously wrong:
The `yml` file would then be located in `/path/to/docker-services-cli/docker_services_cli/docker-services.yml`, but the old way of constructing the path would still give back `/home/mmoser/.local/share/virtualenvs/tu-data-3eTT44w9/lib/python3.8/site-packages/docker_services_cli/docker-services.yml`.

TL;DR: The old way of looking the `yml` file worked for normal installs (`pip install docker-services-cli`), but not for editable installs (`pip install -e ...`).

Reply:
Thanks for taking the time to explain this. PR LGTM!